src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java

```diff
@@ -169,6 +176,67 @@ protected void execute(CommandEvent event)
             }
         }
     }
+
+    public class AppendlistqCmd extends OwnerCommand 
+    {
+        public AppendlistqCmd()
+        {
+            this.name = "appendqueue";
+            this.aliases = new String[]{"addqueue"};
+            this.help = "appends songs in queue to an existing playlist";
+            this.arguments = "<name>";
+            this.guildOnly = false;
+        }
+
+        @Override
+        protected void execute(CommandEvent event) 
+        {
+            String pname = event.getArgs().replaceAll("\\s+", "_");
+            Playlist playlist = bot.getPlaylistLoader().getPlaylist(pname);
+            if(playlist==null)
+                event.replyError(event.getClient().getError()+" Playlist `"+pname+"` doesn't exist!");
```

Comment:
Whoops! This will result in a message like:
> ⚠️⚠️ Playlist `name` doesn't exist!

`event.replyError("text")` adds the emoji to the beginning for you, which is much better than to do `event.reply(event.getClient().getError()+" text")`. This function didnt exist a few years ago, some old code still uses that. 

Nowadays, you just wanna do `event.replyError("text")`.

```suggestion
                event.replyError("Playlist `"+pname+"` doesn't exist!");
```